### DISCUSSION TITLE
Podman 4.5.1, qemu and Mac M1 - the solution

### DISCUSSION THREAD

**Discussion by benfro** [containers/podman#19166] [General]
Hi all,
for a couple of weeks I've been searching the web for the solution to "qemu exited unexpectedly with exit code -1, stderr: qemu-system-x86_64: Unknown Error" on my Apple M1 Max, Mac OS Venture 13.4.1 machine. Podman installed via brew. (Oddly enough my work machine, M2, does NOT have these problems) 

There are fragments of thoughts out there , some say "hey, this worked for me" and then som exotic hack in the deeper intestines and belly of the OS... which probably makes you destroy other things. Then try a couple of them and you'll be happy if your machine ever starts again...

I find this very unsatisfactory. There is clearly a bug, a huge one. Is there any action on it at all? I'm just a developer wanting to use the great tool, not a hacker with clandestine, secret knowledge of how a linux kernel behaves in depth.

There should be some clean, distinct "do this"-list to resolve it.

Any tips about one?

Regards,

**Comment by Luap99** [comment]
> qemu-system-x86_64 

If you are on a M1 you installed the wrong podman version, you need the aarch64 one.

**Reply by joubertredrat** [reply]
But I have a doubt, I installed exactly as mentioned on https://podman.io/docs/installation#macos and I have same issue from him and I didn't found any information in troubleshooting. Do you know if have specific install documentation for macos m1/m2/arm?

**Reply by joubertredrat** [reply]
Hi guys, I solved my problem reinstalling the brew, following the information on this link. https://earthly.dev/blog/homebrew-on-m1/

**Comment by benfro** [comment]
Yes! Thank you, that was the clear explanation I needed. Clearly, the bug is in the Brew install script that STILL installs to intel path.
However, with @joubertredrat good link together with https://stackoverflow.com/questions/66242037/how-to-uninstall-homebrew-of-rosetta-and-install-homebrew-native-on-mac-bigsu-m (because I had tampered with the old install so it wouldn't go away).

Much obliged. Again, this is a Brew install script error - it installs to the Intel path, and then, when you have gotten the new /opt path up - use "arch -arm64 brew install XXX" to make it work.

Thanks again!

PS. Bloody odd that Brew et al. worked flaw- and seamlessly on my work M2 machine without all this monkey business....DS.

**Comment by rhatdan** [comment]
@ashley-cui do we have anything to do with the brew install script?
